src/azure-cli/azure/cli/command_modules/rdbms/_help.py

```diff
@@ -309,6 +309,41 @@
     text: az mariadb server-logs list -g testgroup -s testsvr --max-file-size 30
 """
 
+helps['mariadb server private-endpoint-connection'] = """
+type: group
+short-summary: Manage MariaDB server private endpoint connections.
+"""
+
+helps['mariadb server private-endpoint-connection approve'] = """
+type: command
+short-summary: Approve the specified private endpoint connection associated with a MariaDB server.
+"""
+
+helps['mariadb server private-endpoint-connection reject'] = """
+type: command
+short-summary: Reject the specified private endpoint connection associated with a MariaDB server.
+"""
+
+helps['mariadb server private-endpoint-connection delete'] = """
+type: command
+short-summary: Delete the specified private endpoint connection associated with a MariaDB server.
+"""
+
+helps['mariadb server private-endpoint-connection show'] = """
+type: command
+short-summary: Show details of a private endpoint connection associated with a MariaDB server.
+"""
+
+helps['mariadb server private-link-resource'] = """
+type: group
+short-summary: Manage MariaDB server private link resources.
+"""
+
+helps['mariadb server private-link-resource show'] = """
```

Comment:
Hello. Please use list instead of show. KeyVault command would change in near future too.

Reply:
done